packages/tyxml-ppx/tyxml-ppx.4.5.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "PPX to write TyXML documents with the HTML syntax"
+description: """
+```ocaml
+open Tyxml
+let%html to_ocaml = "<a href='ocaml.org'>OCaml!</a>"
+```
+ 
+The TyXML PPX allow to write TyXML documents using the traditional HTML syntax. 
+It works with textual trees, virtual DOM trees, or any TyXML module.
+"""
+maintainer: ["dev@ocsigen.org"]
+authors: ["The ocsigen team"]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/ocsigen/tyxml"
+doc: "https://ocsigen.org/tyxml/latest/manual/intro"
+bug-reports: "https://github.com/ocsigen/tyxml/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.04"}
+  "tyxml" {version}
+  "tyxml-syntax" {version}
+  "alcotest" {with-test}
+  "markup" {>= "0.7.2"}
+  "ppxlib"
```

Comment:
I highly suspect this requires a lower-bound constraint (at least >= 0.9 or something like that)